testing/kuttl/e2e/pgbackrest-init/00-assert.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: init-pgbackrest
+status:
+  pgbackrest:
+    repoHost:
+      apiVersion: apps/v1
+      kind: StatefulSet
+      ready: true
+    repos:
+# Assert that the status has the two repos, with only the first having the `replicaCreateBackupComplete` field
+    - bound: true
+      name: repo1
+      replicaCreateBackupComplete: true
+      stanzaCreated: true
+    - bound: true
+      name: repo2
+      stanzaCreated: true
```

Comment:
Does this assertion work here? I think that `replicaCreateBackupComplete` could be set on `repo2` here and we just wouldn't look at that field in the test :thinking:

Reply:
When I first create a cluster with two repos, this is the `status.pgbackrest` -- I wondered at first if the second repo would ever have that field set, but it looks like that field is only for replica jobs (https://github.com/CrunchyData/postgres-operator/blob/master/internal/controller/postgrescluster/pgbackrest.go#L2086) and not for manual backup jobs

Reply:
Ok, wfm. I was just thinking that if the second repo did have the status our test wouldn't care.